explore-analyze/elastic-inference.md

```diff
@@ -7,7 +7,13 @@ navigation_title: Elastic Inference
 
 # Elastic {{infer-cap}}
 
-There are several ways to perform {{infer}} in the {{stack}}. This page provides a brief overview of the different methods:
+## Overview
 
+{{infer-cap}} is a process of using a {{ml}} trained model to make predictions or operations - such as text embedding, or reranking - on your data.
+You can use {{infer}} during ingest time (for example, to create embeddings from textual data you ingest) or search time (to perform [semantic search](/solutions/search/semantic-search.md)).
```

Comment:
nit: you need to embed texts at ingest time order to enable semantic search. This makes it sounds like I could _only_ use inference at search time and that this would be enough.